app/views/building_map/index.html.erb

```diff
@@ -38,6 +38,13 @@
         </div>
       </div>
     </div>
+    <div class="d-grid gap-2 col-6">
+       <div class="form-check form-switch ms-4">
+         <input class="form-check-input" type="checkbox" role="switch" id="tracking_switch" />
+          <%# TODO: second switch for deciding whether or not to send the location to the server %>
```

Comment:
Since this PR is taking way longer than it was supposed to, we think that we should merge with basic functionality (the API for getting the position on the `User` model) so that others can start working on follow-up issues while we plug the holes we left in a new PR. These changes include

- The UI to distinguish between showing the location on the map and sharing with the server
- A mechanism that actually deletes old user positions / deletes them when the site is left
- (maybe tests for the functionality, but it probably won't work, see: all of this PR)
- ... maybe things that I forgot to write right now :^)

Anyone against that plan?